dhcpv6/async/client_test.go

```diff
@@ -26,7 +26,7 @@ func solicit(input string) (*dhcpv6.Message, error) {
 		Time:          dhcpv6.GetTime(),
 		LinkLayerAddr: mac,
 	}
-	return dhcpv6.NewSolicitWithCID(duid)
+	return dhcpv6.NewSolicitWithCID(duid, mac, dhcpv6.WithIAID([4]byte{1, 2, 3, 4}))
```

Comment:
How about using the duid to generate the IAID instead of having to pass the mac?

Reply:
I am using the same approach as dhclient, I'd rather stick with it. This can be changed with a modifier if necessary

Reply:
The DUID has the MAC inside it so it feels redundant.